src/include/platform/ConnectivityManager.h

```diff
@@ -171,6 +174,16 @@ class ConnectivityManager
     bool IsWiFiStationProvisioned();
     void ClearWiFiStationProvision();
     CHIP_ERROR GetAndLogWiFiStatsCounters();
+#if CHIP_DEVICE_CONFIG_ENABLE_WPA && CHIP_DEVICE_CONFIG_ENABLE_WIFIPAF
+    CHIP_ERROR SetWiFiPAFAdvertisingEnabled(bool val);
+
+    typedef void (*OnConnectionCompleteFunct)(void * appState);
+    typedef void (*OnConnectionErrorFunct)(void * appState, CHIP_ERROR err);
+    CHIP_ERROR WiFiPAFConnect(void * appState, OnConnectionCompleteFunct onSuccess, OnConnectionErrorFunct onError);
```

Comment:
Does this API assume there can only be a single PAF connection ongoing at a time? We shouldn't be repeating the mistakes made with the BLE implementation.

Reply:
Yes, I assume there is a single PAF at a time for now. We will check if more PAF concurrent connection are available.
BTW, could you let me know what's the mistake of the BLE implementation? Thank you

Reply:
The mistake in the BLE implementation was baking the assumption that we can only be connected to a single device over BLE at a time into the internal APIs. We will need the PAF APIs to cleanly support discovering multiple commissionable devices at the same time, and send and receive follow-up packets to/from one or more of them.

Reply:
May we support one single device at a time for now?

We think more tasks are required to support multiple sessions concurrently:
1. We think the CHIPDeviceController may need to change to support multiple PASEConnection concurrently
2. The chip-tool parameters should be changed to pair with multiple nodes concurrently
3. Maybe the manual_pairing_code and qrcode need to be changed as well.
Are these ready that we can test if supporting multiple sessions concurrently? If not, may we add it in the todo tasks in the next PR? If yes, could you suggest more information that we can reference? Thank you.

Reply:
CHIPDeviceController would definitely need to change.

chip-tool might keep supporting only one thing at a time.

But the APIs for PAF that CHIPDeviceController uses should be done in such a way that they can handle more than one thing concurrently, ideally.

Reply:
So, may we keep using it for now?

Reply:
Please file an issue to track this follow-up work.